testdata/messages/integration/test_proj_with_vulnerability_simplified.md

```diff
@@ -5,26 +5,60 @@
 **🚨 Frogbot scanned this pull request and found the below:**
 
 
+
 ---
-## 📦 Vulnerable Dependencies
+## 📗 Scan Summary
 
 ---
+- Frogbot scanned for Vulnerabilities and found 1 issues
+
+| Scan Category                | Status                  | Security Issues                  |
+| --------------------- | :-----------------------------------: | ----------------------------------- |
+| **Software Composition Analysis** | ✅ Done | 1 Issues Found: ❗️ 1 Critical |
+| **Contextual Analysis** | ✅ Done | - |
+| **Static Application Security Testing (SAST)** | ✅ Done | Not Found |
+| **Secrets** | ✅ Done | - |
+| **Infrastructure as Code (IaC)** | ✅ Done | Not Found |
 
 
 ---
-### ✍️ Summary
+### 📦 Vulnerable Dependencies
 
 ---
-| SEVERITY                | CONTEXTUAL ANALYSIS                  | DIRECT DEPENDENCIES                  | IMPACTED DEPENDENCY                  | FIXED VERSIONS                  | CVES                  |
+
+| Severity                | ID                  | Contextual Analysis                  | Direct Dependencies                  | Impacted Dependency                  | Fixed Versions                  |
 | :---------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: |
-| Critical | Not Applicable | minimist:1.2.5 | minimist 1.2.5 | [0.2.4], [1.2.6] | CVE-2021-44906 |
+| Critical | CVE-2021-44906 | Not Applicable | minimist:1.2.5 | minimist 1.2.5 | [0.2.4], [1.2.6] |
+
+
+---
+### 🔖 Details
```

Comment:
Is the empty section under 'Details' intended? or we just have 2 empty lines there?
This also happens in the other md files